### PR TITLE
Layout and responsiveness changes

### PR DESCRIPTION
**Pull request by Gujansko**
- Each section content has it's own height and scrollbar
- Deleted body constantly appearing scrollbar
- Added content to consequences section
- Added background
- Added links to social medias
Examples of new sections looks:
image
image

### CODE REVIEW

File: src/components/PaymentsForm/PaymentForm.tsx [hunk 1]
```diff
@@ -61,37 +61,41 @@ const PaymentsForm = () => {
...
+                  <div className="w-1/2 max-xl:w-full">
+                    <FormField
+                      label={languageMode == "english" ? "Surname" : "Nazwisko"}
+                      isRequired={true}
+                      minLength={3}
+                      maxLength={30}
+                      registerName="surname"
+                    />
+                  </div>
                 </div>
```
Comment (Misiek22113) [inline src/components/PaymentsForm/PaymentForm.tsx:84]: To be honest i do not understand why there are these third div's(the ones that wraps FormFIeld).

**Review by Misiek22113 — APPROVED** [review]

File: src/app/globals.css [hunk 2]
```diff
@@ -74,6 +73,33 @@
...
+  scroll-behavior: smooth;
+}
+
+main {
+  height: calc(100svh - var(--nav-height) - 2px);
+}
+
+.section-content-wrapper {
+  height: calc(
+    100svh - var(--nav-height) - var(--section-headers-height) - 2px
```
[the remark this answers is not in the document]
Comment (Gujansko) [reply src/app/globals.css:99]: Looks pretty straight forward to me. We have 3 elements on this page and I want the third element to take space left after we subtract the space that first two elements take from overall available space. This is mostly done to make sections their own size not align all sections to the size of biggest one. That would create a lot of blank space for smaller sections

File: src/components/PaymentsForm/PaymentForm.tsx (hunk 1 above)
Comment (Gujansko) [reply src/components/PaymentsForm/PaymentForm.tsx:84]: This is to manipulate the space input takes. If form components would have className I would add this styling to them. We had a problem where these two fields were a lot smaller when going to mobile than other fields below it. That takes care of it

Comment (Gujansko) [reply src/components/PaymentsForm/PaymentForm.tsx:84]: Before
image
After
image
I will add custom className to field to make it look better

**Pull request merged**